data_hub_api/docmaps/v2/docmap_typing.py

```diff
@@ -43,6 +43,19 @@
     }
 )
 
+DocmapPublishedElifeManuscriptOutput = TypedDict(
+    'DocmapPublishedElifeManuscriptOutput',
+    {
+        'type': str,
+        'published': str,
```

Comment:
It seems at the moment, the `published` date is optional? (Until you implemented the rule to not include the published step)